docs/src/index.md

```diff
@@ -1,51 +1,273 @@
 # FiniteDifferences.jl: Finite Difference Methods
```

Comment:
I don't care for the copy paste.
One bit will get outdated.

Can we do the copy step during the build of the docs?
In `docs/make.jl`

Though it's not changed by this PR so maybe van just open an issue?
And make sure this PR stays right

Reply:
I've added the copy step in `docs/make.jl`. That's indeed more convenient.